app/controllers/api/visits_controller.rb

```diff
@@ -16,6 +18,10 @@ def update
 
     private
 
+    def check_access_to_visit(visit)
```

Comment:
Could those methods exist elsewhere than the controllers ? Like a dedicated helper object ?

Reply:
Maybe in the application controller ?